app/views/experience_stages/show.html.erb

```diff
@@ -91,15 +91,15 @@
               </div>
             </div>
 
-            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=show_all'>
+            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=false'>
```

Comment:
Gah ok - let's try to see if we can get the order stuff working first, and then we can revisit this. Angular can sometimes be a pain in the ass with this kind of stuff...